rmf_site_format/src/workcell.rs

```diff
@@ -280,6 +280,137 @@ pub struct Workcell {
     // TODO(luca) Joints
 }
 
+impl From<Pose> for urdf_rs::Pose {
+    fn from(pose: Pose) -> Self {
+        urdf_rs::Pose {
+            rpy: match pose.rot {
+                Rotation::EulerExtrinsicXYZ(arr) => urdf_rs::Vec3(arr.map(|v| match v {
+                    Angle::Rad(v) => v as f64,
+                    Angle::Deg(v) => v.to_radians() as f64,
+                })),
+                Rotation::Yaw(v) => match v {
+                    Angle::Rad(v) => urdf_rs::Vec3([0.0, 0.0, v as f64]),
+                    Angle::Deg(v) => urdf_rs::Vec3([0.0, 0.0, v.to_radians() as f64]),
+                },
+                _ => todo!("Unsupported rotation type for conversion to urdf pose"),
```

Comment:
This will cause the code to panic if `pose.rot` is a `Rotation::Quat`. That's a big problem since it would be perfectly reasonable for someone to use quaternions to describe a pose.

Another issue with this pattern is if we ever add another variant to `Rotation` then we'll never notice that we need to update this pattern matching block. We should prefer for this block to produce a compilation error if we ever add another `Rotation` variant.

So let's get rid of this line and add a pattern matcher for `Rotation::Quat(_)`. We can draw inspiration from the implementation [here](https://github.com/open-rmf/rmf_site/blob/d2fb61f8785e66021b56286fb64b6fda454f9ffe/rmf_site_format/src/misc.rs#L261-L270), except I believe that was implemented before we accepted `glam` as a dependency. `glam` is the crate that bevy uses for math, and it's [now a dependency for `rmf_site_format`](https://github.com/open-rmf/rmf_site/blob/d2fb61f8785e66021b56286fb64b6fda454f9ffe/rmf_site_format/Cargo.toml#L17) so we can feel free to use glam for things like this.

Another tip is that we don't need to do any pattern matching for `Angle` because `Angle` has a [`radians()`](https://github.com/open-rmf/rmf_site/blob/d2fb61f8785e66021b56286fb64b6fda454f9ffe/rmf_site_format/src/misc.rs#L146-L151) method. E.g. consider this alternative for the first matching arm:
```
Rotation::EulerExtrinsicXYZ(arr) => urdf_rs::Vec3(arr.map(|v| v.radians())),
```

Reply:
Done in dfbf3d3.